sysinit/coqargs.ml

```diff
@@ -442,15 +442,6 @@ let prelude_data = "Prelude", Some "Coq", Some Lib.Import
 let injection_commands opts =
   if opts.pre.load_init then RequireInjection prelude_data :: opts.pre.injections else opts.pre.injections
 
-let build_load_path opts =
```

Comment:
Just curious, if I read this correctly, it seems to me you did inline this function in `coqinit`, why?

The idea of the function was for it to be used by coq-serapi and jscoq (and other toplevels). But indeed, the name should be `load_paths_from_args` to make that clearer.

So not a big deal but it seemed a good idea to me to keep the separation between the loadpaths that come from args and the rest.

Reply:
The separation made it harder to work with IMO.
There is no way for me to know what plugins outside CI depend on, they'll have to adapt on their own.

Reply:
Yeah I lost track, so I guess there was a version of the PR that did modify it, now it is just inlined so in a sense that change is noise. I don't mind it.